lib/utils/fs.js

```diff
@@ -0,0 +1,25 @@
+/*
+getCWD kindly taken from henrikdahl's plugin hyper statusline
+
+https://github.com/henrikdahl/hyper-statusline/blob/master/index.js
+*/
+
+import {execSync} from 'child_process';
+
+export default function getCWD(pid, action) {
+  let cwd;
+  if (process.platform == 'win32') {
+    let directoryRegex = /([a-zA-Z]+)/im;
+    if (action && action.data) {
+      let path = directoryRegex.exec(action.data);
+      if (path) cwd = path[0];
+    }
+  } else {
+    cwd = execSync(`lsof -p ${pid} | awk '$4=="cwd"' | tr -s ' ' | cut -d ' ' -f9-`, {encoding: 'utf8'});
```

Comment:
Is this the best way to get the current CWD? 🤔 I wonder if there is an async way of doing it so we don't lock up the app...

Reply:
@albinekb There are async methods of running child_processes, but I thought in Redux making stuff async is frowned upon (still learning, I've heard of thunks being used for async actions). That's why I only have this function run when the user hits the enter key.

Reply:
It will be executed each time that '\n' is output in term. Do not execute `yes` in your fork 😁